src/images/maps/gg_map.R

```diff
@@ -13,8 +13,8 @@ box::use(../plots/theme_signals)
 #' @returns ggplot object
 #'
 #' @export
-gg_map <- function(iso3) {
+gg_map <- function(iso3, ...) {
```

Comment:
Curious, is the `...` an R convention (like Python's `kwargs`)?

Reply:
Yeah! Pretty similar. Although it can also accept positional argument arguments, not just named arguments. So you can do something like `a <- function(...) sum(...)` and just pass everything on. But named arguments are the typical use, where you are wrapping one function that you want users to have the option to pass arguments to using the ellipsis!